dradis/dradis-ce
# Add textile comments filter and comments formatter

### Spec
Mentions in comments are borked when there is a period `.` present in a users email. This is because the textile filter evaluates `@not.code@asdf` as an inline code *@not.code@asdf*.

**Proposed Solution**
Make inline code no-op for textile in comments for now.

### How to test:
1. Add a user with email: `test.user@gmail.com`
2. Mention that user
3. Confirm that the user's avatar correctly displays
